composer.json

```diff
@@ -21,7 +21,7 @@
         "silverstripe/admin": "^1.7@dev",
         "silverstripe/versioned": "^1.7@dev",
         "silverstripe/versioned-admin": "^1.7@dev",
-        "silverstripe/graphql": "^3.4",
+        "silverstripe/graphql": "^3.4 || 4.x-dev",
```

Comment:
Would it be OK to make this change in `4.5` branch of elemental instead to increase test coverage? Or is to too much of change?

Reply:
True, GraphQL v4 support was merged before the 4.5.0 stable release. So this would allow early adopters to run a non-stable version of GraphQL while staying on a stable version of Elemental. But then again, I don't really want to encourage that since we might need to further update elemental to keep up with changes in GraphQL v4. So on balance, I think the change is targeted correctly at `4`